e2e/testcases/reconciler_finalizer_test.go

```diff
@@ -757,6 +760,257 @@ func deleteObject(nt *nomostest.NT, obj client.Object) error {
 	return nil
 }
 
+// TestReconcilerFinalizer_ResourceGroupMetricsReset tests that when a ResourceGroup
+// is deleted, all resource-related metrics are reset to 0.
+func TestReconcilerFinalizer_ResourceGroupMetricsReset(t *testing.T) {
+	nt := nomostest.New(t, nomostesting.MultiRepos)
```

Comment:
nit: suggest using `Unstructured` repository for testing instead of hierarchical

Reply:
Done